src/components/pages/CitySearch/Index.js

```diff
@@ -1,31 +1,50 @@
 import React from 'react';
 
 import logic from './logic.js';
+import dummyData from '../../../utils/dummyData';
 // import map from 'map';
```

Comment:
Please remove dead code.

Reply:
Yeah, I'll remove this file since there's no reason to push something I'm using to develop with.